spec.html

```diff
@@ -32086,27 +32086,128 @@ <h1>Week Day</h1>
         <p>A weekday value of *+0*<sub>𝔽</sub> specifies Sunday; *1*<sub>𝔽</sub> specifies Monday; *2*<sub>𝔽</sub> specifies Tuesday; *3*<sub>𝔽</sub> specifies Wednesday; *4*<sub>𝔽</sub> specifies Thursday; *5*<sub>𝔽</sub> specifies Friday; and *6*<sub>𝔽</sub> specifies Saturday. Note that <emu-eqn>WeekDay(*+0*<sub>𝔽</sub>) = *4*<sub>𝔽</sub></emu-eqn>, corresponding to Thursday, 1 January 1970.</p>
       </emu-clause>
 
-      <emu-clause id="sec-local-time-zone-adjustment" type="implementation-defined abstract operation">
+      <emu-clause id="sec-roundtowardszero" type="abstract operation">
         <h1>
-          LocalTZA (
-            _t_: a Number,
-            _isUTC_: a Boolean,
-          ): an integral Number
+          RoundTowardsZero (
```

Comment:
It _is_ strange to see a general mathematical RoundTowardsZero operation defined under [Date Objects](https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-date-objects); I think it would make more sense in [Mathematical Operations](https://tc39.es/ecma262/multipage/notational-conventions.html#sec-mathematical-operations) (but I don't have an opinion on keeping it an abstract operation vs. a mathematical function such as "truncate(_x_) removes the fractional part of _x_ by rounding towards zero, producing -floor(-_x_) if _x_ &lt; 0 and otherwise producing floor(_x_)").

Reply:
If we are moving it to Mathematical Operations, then making it a mathematical function seems better to me. (There are no other abstract operations in that section.)